vue/src/vueTextMask.js

```diff
@@ -1,24 +1,19 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
 
-export default {
-  textMaskInputElement: null,
-
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
-  },
+let textMaskInputElement = null
```

Comment:
Keeping this outside means there can only be one `textMaskInputElement` per application, which is not what we want.